apps/portals/adknowledgeportal/src/config/resources.ts

```diff
@@ -24,3 +26,5 @@ export const targetEnablingResourcesDetailsPageSql =
   'select * from syn26146692 WHERE `isPublic` = true'
 export const cavaticaConnectAccountURL =
   'https://help.adknowledgeportal.org/apd/Analysis-Platforms.3088482318.html'
+
+export const enabledAnalysisPlatforms: ExternalAnalysisPlatform[] = ['cavatica']
```

Comment:
Store in a constant the platforms we let a portal use. For the portals that support it, these will just be Cavatica for now.